### PR TITLE
Add install notes to README

### PR DESCRIPTION
**Pull request by mrecachinas**
Add how to install and execute scripts to this skeleton

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,76 @@
...
+or
+
+```
+wget -qO- https://raw.githubusercontent.com/creationix/nvm/v0.30.1/install.sh | bash
+```
+
+Once `nvm` is installed, you can install the latest release by simply running
+
+```
+nvm install 5.0
```
Comment (Kevin-K) [inline README.md]: Change to `nvm install stable`

File: README.md [hunk 2]
```diff
@@ -1,2 +1,76 @@
 # skeleton
 A javascript skeleton project utilizing industry popular tooling.
+
+## Installation
+
+### Installing Required Packages
```
Comment (Kevin-K) [inline README.md]: Can you split this between 2 sections:
1) Environment Setup
2) Project Installation

To distinguish between (1) being a set up once for the environment and (2) being for each individual project

**Comment by Kevin-K** [comment]
:+1: squash to 1 commit and I'll merge it.

**Comment by mrecachinas** [comment]
Done :muscle: :ok_hand:

**Pull request merged**